README.md

```diff
@@ -119,3 +118,5 @@ Current projects utilizing this or modified versions of this library:
 
 [gitter]: https://gitter.im/CoraleStudios/Colore?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge
 [gitterbadge]: https://badges.gitter.im/Join%20Chat.svg
+
+[colorelogo]: https://raw.githubusercontent.com/CoraleStudios/Colore/gh-pages/colore.png
```

Comment:
Image assets were moved into their own directory so this should be changed to point to: `https://raw.githubusercontent.com/CoraleStudios/Colore/gh-pages/img/colore.png`.